test/download.spec.js

```diff
@@ -83,6 +83,19 @@ describe('Download', function() {
     expect(fs.readFileSync(path).toString()).toBe('Hello world');
     await page.close();
   })
+  it(`should report download path within page.on('download', …) handler for Blobs`, async({browser, server}) => {
+    const page = await browser.newPage({ acceptDownloads: true });
+    const onDownloadPathPath = new Promise((res) => {
```

Comment:
nit: onDownloadPathPath -> onDownloadPath

Reply:
🤦 Oops! Fixed the `PathPath` from my previous MR as well.